src/dlgTriggerEditor.cpp

```diff
@@ -7888,18 +7859,41 @@ void dlgTriggerEditor::slot_colorizeTriggerSetBgColor()
         return;
     }
 
-    auto color = QColorDialog::getColor(parseButtonStyleSheetColors(mpTriggersMainArea->pushButtonBgColor->styleSheet()),
+    auto color = QColorDialog::getColor(QColor(mpTriggersMainArea->pushButtonBgColor->property(cButtonBaseColor).toString()),
                                         this,
                                         tr("Select background color to apply to matches"));
     if (color.isValid()) {
         mpTriggersMainArea->pushButtonBgColor->setStyleSheet(generateButtonStyleSheet(color));
+        mpTriggersMainArea->pushButtonBgColor->setProperty(cButtonBaseColor, color.name());
     }
 }
 
 void dlgTriggerEditor::slot_soundTrigger()
 {
-    QString fileName = QFileDialog::getOpenFileName(this, tr("choose sound file"), QDir::homePath(), tr("*"));
-    mpTriggersMainArea->lineEdit_soundFile->setText(fileName);
+    // Use the existing path/filename if it is not empty, otherwise start in
+    // profile home directory:
+    QString fileName = QFileDialog::getOpenFileName(this,
+                                                    tr("Choose sound file"),
+                                                    mpTriggersMainArea->lineEdit_soundFile->text().isEmpty()
+                                                    ? mudlet::getMudletPath(mudlet::profileHomePath, mpHost->getName())
+                                                    : mpTriggersMainArea->lineEdit_soundFile->text(),
+                                                    tr("Audio files(*.aac *.mp3 *.mp4a *.oga *.ogg *.pcm *.wav *.wma);;"
```

Comment:
`QMediaPlayer` is quite adept at playing things and it possibly might depend on the underlying system libraries that it can dynamically link to - so I do not have any reason to not believe Mudlet can handle them - it is just that adding the filters makes it easier if the user is looking for a particular file if they can hide files that are definitely NOT audio files...